feature/intern/src/main/java/com/terning/feature/intern/component/InternCompanyInfo.kt

```diff
@@ -36,9 +37,14 @@ fun InternCompanyInfo(
     company: String,
     companyCategory: String,
 ) {
+    val image = if (companyImage == "") {
+        ic_terning_intern_item_image_loading_128
+    } else {
+        companyImage
+    }
```

Comment:
```suggestion
    val image = companyImage.ifBlank {
        ic_terning_intern_item_image_loading_128
    } 
```
`ifBlank`라는걸 사용해서 빈 문자열인 경우 다른 값을 반환할 수 있어요! 사용하면 좋을 것 같네요!